src/main/kotlin/com/angels/challenge/entity/SeasonEntity.kt

```diff
@@ -0,0 +1,16 @@
+package com.angels.challenge.entity
+
+import java.io.Serializable
+import javax.persistence.*
+
+
+@Entity
+@Table(name="season")
+class SeasonEntity() : Serializable {
+    @Id
+    @Column(name = "id")
+    var id: String? = null
```

Comment:
Why is the id nullable?

Reply:
being the id should not be null, I should have left it as 
```suggestion
    lateinit var id: String
```
instead 🤔

Reply:
👍